src/simplification/SimplifyExtend.ts

```diff
@@ -0,0 +1,30 @@
+import { CommonModel } from "models";
+import { Schema } from "models/Schema";
+import Simplifier from "./Simplifier";
+type output = {newModels: CommonModel[] | undefined; extendingSchemas: string[] | undefined};
```

Comment:
I know that at the moment it's not important but then we should make something like 

```ts
type CommonOutput = { newModels?: CommonModel[] }
type ExtendOutput = CommonOutput & { extendingSchemas?: string[] }
```